systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/controllers/ConfigMapUtils.java

```diff
@@ -48,7 +49,7 @@ public static void waitForConfigMapLabelsChange(String namespaceName, String con
         }
     }
 
-    public static void waitForConfigMapLabelsDeletion(String namespaceName, String configMapName, String... labelKeys) {
+    public static void waitForConfigMapLabelsDeletion(String namespaceName, String configMapName, Collection<String> labelKeys) {
```

Comment:
Why is this needed? I can't see the benefit - you are not using anything from the `Collection` - or is there something I don't see?

Reply:
I removed this method same as the one referenced above.